inventories/fun.yaml

```
@@ -0,0 +1,60 @@
+all:
+  vars:
+    ansible_user: root
```

Wouldn't it be preferable?

```suggestion
    ansible_become: yes
```